.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
We're not mentioning anywhere that 18 is recommended for development. Should we? Maybe in the pnpm PR?

Reply:
Do we suggest `nvm`? Have not looked into the differences, but I have been using [`n`](https://github.com/tj/n) without issues. Perhaps to consult with the team.

Reply:
i've used both `n` and `nvm`, and they are both good for the job. but yes, we should ask the team first. I'll remove this file.